### PR TITLE
chore: swap out twitter logo

### PR DESCRIPTION
**Pull request by alexanderleegs**
Swaps the twitter logo out for the x logo. Note that this does not use sgds or boxicons, because neither have the x logo - it is inserted as an svg instead.

Test site:
https://staging.duyfy15grdtiq.amplifyapp.com/

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -384,7 +384,7 @@ <h5 class="sub-header has-text-white">
                     {%- assign url_input = social.twitter -%}
                     {%- include functions/external_url.html -%}
                     <a {{anchor}} class="social-link padding--right" title="Twitter">
-                        <span class="sgds-icon sgds-icon-twitter is-size-4"></span>
+                        <img class="social-logo" src="/assets/img/x_logo.svg" alt="Open Government Products Logo">
```
Comment (dcshzj) [inline _includes/footer.html]: issue: Sadly OGP doesn't own twitter

File: _includes/footer.html [hunk 2]
```diff
@@ -228,7 +228,7 @@ <h5 class="sub-header has-text-white">
                         {%- assign url_input = social.twitter -%}
                         {%- include functions/external_url.html -%}
                         <a {{anchor}} class="social-link padding--left padding--bottom is-inline-block">
-                            <span class="sgds-icon sgds-icon-twitter is-size-4" title="Twitter"></span>
+                            <img class="social-logo" src="/assets/img/x_logo.svg" alt="Open Government Products Logo">
```
Comment (dcshzj) [inline _includes/footer.html]: issue: same here

File: _includes/footer.html (hunk 1 above)
Comment (kishore03109) [reply _includes/footer.html]: LOL

Comment (alexanderleegs) [reply _includes/footer.html]: oops thanks for catching https://github.com/isomerpages/isomerpages-template/pull/362/commits/1168a34d5dff096e0028dfbb2a651718952bc249

File: _includes/footer.html (hunk 2 above)
Comment (alexanderleegs) [reply _includes/footer.html]: https://github.com/isomerpages/isomerpages-template/pull/362/commits/1168a34d5dff096e0028dfbb2a651718952bc249

**Review by dcshzj — APPROVED** [review]
LGTM!

**Review by QiluXie — APPROVED** [review]
lgtm

File: _includes/footer.html (hunk 1 above)
Comment (QiluXie) [reply _includes/footer.html]: nice catch lol

**Pull request merged**